src/pages/omOss.tsx

```diff
@@ -0,0 +1,253 @@
+        <div className="px-[100px] mt-36 bg-[url('/img/grayKth.jpg')] bg-cover bg-no-repeat bg-center h-[600px] w-full flex items-center justify-center gap-5">
+          <div className="flex flex-col">
+            <div className="flex flex-row items-center justify-center mb-10">
+              <div className="">
+                <h2 className="text-cerise text-xl font-medium">
+                  {" "}
+                  {t.aboutUs.info1}
+                </h2>
+                <p className="text-white text-lg">{t.aboutUs.paragraph1} </p>
+              </div>
+            </div>
+            <div className="flex flex-row">
+              <div className="">
+                <h2 className="text-cerise text-xl font-medium">
+                  {t.aboutUs.info2}
+                </h2>
+                <p className="text-white text-lg">{t.aboutUs.paragraph2}</p>
+              </div>
+
+              <div className="">
+                <h2 className="text-cerise text-xl  font-medium">
+                  {t.aboutUs.info3}
+                </h2>
+                <p className="text-white text-lg">{t.aboutUs.paragraph3}</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      {/*
+        Phone version version
+      */}
+      <div className="flex flex-row items-center justify-center lg:hidden">
+        <div className="px-[100px] mt-36 bg-[url('/img/grayKth.jpg')] bg-cover bg-no-repeat bg-center h-[150px]">
+          
+        </div>
+      </div>
+      <div className="flex flex-col px-[50px] mt-[100px] mb-[100px] lg:hidden">
+            <div className="flex flex-row items-center justify-center mb-10">
+              <div className="">
+                <h2 className="text-cerise text-xs font-medium">
+                  {" "}
+                  {t.aboutUs.info1}
+                </h2>
+                <p className="text-white text-xs">{t.aboutUs.paragraph1} </p>
+              </div>
+            </div>
+            <div className="flex flex-row">
+              <div className="">
+                <h2 className="text-cerise text-xs font-medium">
+                  {t.aboutUs.info2}
+                </h2>
+                <p className="text-white text-xs">{t.aboutUs.paragraph2}</p>
+              </div>
+
+              <div className="">
+                <h2 className="text-cerise text-xs  font-medium">
+                  {t.aboutUs.info3}
+                </h2>
+                <p className="text-white text-xs">{t.aboutUs.paragraph3}</p>
+              </div>
+            </div>
+          </div>
+
+      <div className="flex flex-row items-center justify-center px-[400px] hidden lg:block">
+        <div className="px-[100px] mt-[100px] bg-[url('/img/groupPictures/projectGroup.jpg')] bg-cover bg-no-repeat bg-center h-[400px] w-full flex flex-row items-center justify-center">
+          <p className="text-cerise text-8xl">{t.aboutUs.subHeader}</p>
+        </div>
+      </div>
```

Comment:
The "Meet the team" text should probably expand horizontally to fit everything in one line
<img width="1281" alt="image" src="https://github.com/datasektionen/ddagen/assets/66337956/b4e176db-70fd-4b7a-8691-8e37895e3a3e">